File: insertionSort/insertionSort.py
def insertion_sort( arr ):
    for index in range(1, len(arr)):
        position = index
        while position > 0 and arr[position - 1] > arr[position]:
            swap = arr[position]
            arr[position] = arr[position - 1]
            arr[position - 1] = swap
            position = position - 1

    return arr

File: insertionSort/insertionSort.js
'use strict';

const insertionSort = ( arr ) => {
  for ( let i = 0; i < arr.length; i++ ) {
    let j = i;
    while ( j > 0 && arr[ j ] < arr[ j - 1 ] ) {
      let swap = arr[ j - 1 ];
      arr[ j - 1 ] = arr[ j ];
      arr[ j ] = swap;
      j = j - 1;
    }
  }
  return arr;
};

module.exports = insertionSort;
